src/Http/Error.php

```diff
@@ -45,13 +45,33 @@ public function __construct(
         string $message = ''
     ) {
 
+        $factory = \NDB_Factory::singleton();
+
         $uri     = $request->getURI();
         $baseurl = $uri->getScheme() .'://'. $uri->getAuthority();
-
+        // Admistrator email.
+        $contact  = $request
+            ->getAttribute('loris')
+            ->getConfiguration()
+            ->getSetting('mail')['From'];
         $tpl_data = array(
                      'message' => $message,
                      'baseurl' => $baseurl,
+                     'contact' => $contact,
                     );
+        // Add issue tracker data if the error is encountered by an authenticated
+        // user with the correct permissions.
+        $user = $request->getAttribute('user');
+        if (! $user instanceof \LORIS\AnonymousUser) {
```

Comment:
This if logic here is wrong. canReport should be based on hasAnyPermission regardless of the type of user (AnonymousUsers won't have the permission, and if LORIS is intentionally set up so that they do it shouldn't be bypassed.), and issueTrackerURL should be based on whether canReport is true.

Reply:
OK I can remove the if statement.

I'm not really sure what you mean by "issueTrackerURL should be based on whether canReport is true."

Reply:
The other template variable conditionally being set in the if statement

```
  $tpl_data['issueTrackerURL'] = '/issue_tracker/issue/new';
```

The condition for that being set should be `if ($canReport === true)`, not `if (! $user instanceof \LORIS\AnonymousUser)`.. though it also doesn't matter if you just unconditionally set it, since the template won't use it if it doesn't need it.